vizro-core/mkdocs.yml

```diff
@@ -2,6 +2,9 @@ site_name: Vizro
 site_url: https://mckinsey.github.io/vizro/
 nav:
   - Vizro: index.md
+  - Quickstart:
```

Comment:
Would still stick with `Tutorials`

Reply:
I actually liked the Quickstart description better, @Joseph-Perkins any preference?

Reply:
I think we should follow https://diataxis.fr/ here. 

Arguably the current tutorials are not even "quick" starts, if you wanted to start real quick, you would just go to a user guide.

In the future we will definitely have more tutorials, some of them will be quite advanced, and certainly not quick. And we ideally do not want to change our docs so fundamentally then

Reply:
Where does it say that Quickstart is not suitable as a description? 

I kind of liked Dash documentation here: https://dash.plotly.com/
In their Quickstart section, they also have a small example and then a more thorough 20-minute tutorial.

Reply:
That's why I would leave the second card title as Tutorial, as this contains our extensive example. I don't think it takes an hour and even if it's still fast compared to building an app with all these features from scratch?

![Screenshot 2023-09-13 at 14 56 21](https://github.com/mckinsey/vizro/assets/90609403/82d8226f-d90d-4fbe-812f-588b8a3310f4)

Either way, I am fine with your proposal if J. also prefers it 😄

Reply:
Should we not maybe name the tutorials themselves then `Quickstart` and `Explore Vizro` 🤔 ?

Reply:
Or the main section is called **Get started**, and the sub-sections Quickstart and Tutorial? Would be in line with other packages e.g. 
- https://pandas.pydata.org/docs/index.html
- https://panel.holoviz.org/getting_started/index.html

Reply:
I don't mind being generic in the title, there is also a description explicitly stating "Follow our step-by-step tutorial to learn about all the available features". 

So personally I prefer **Tutorial** and then eventually rephrase the description to `"Follow our step-by-step tutorial to learn about all our core features"`. I don't mind the current title "Explore Vizro" either though.

Reply:
Not necessary anymore as I've removed the card